File: php/src/Model/PaperSize.php
<?php

declare(strict_types=1);

namespace Dokmatiq\DocGen\Model;

enum PaperSize: string
{
    case A4 = 'A4';
    case A3 = 'A3';
    case A5 = 'A5';
    case LETTER = 'LETTER';
    case LEGAL = 'LEGAL';
}

File: java/src/main/java/com/dokmatiq/docgen/model/PaperSize.java
package com.dokmatiq.docgen.model;

import com.fasterxml.jackson.annotation.JsonValue;

/** Standard paper sizes. */
public enum PaperSize {
    A4("A4"), A3("A3"), A5("A5"), LETTER("LETTER"), LEGAL("LEGAL");

    private final String value;

    PaperSize(String value) { this.value = value; }

    @JsonValue
    public String getValue() { return value; }
}
